source/widgets/library_widget.py

```diff
@@ -688,8 +688,19 @@ def make_portable_path(self):
         version = self.build_info.subversion.rsplit(".", 1)[0]
 
         if version >= "4.2":
-            folder_name = "portable"
-            config_path = self.link / folder_name
+            # Check for portable folder first (standard Blender 4.2+)
+            portable_path = self.link / "portable"
+            if portable_path.is_dir():
+                return portable_path
+
+            # Check for version folder (X.Y format) used by bforartists
+            if get_platform() == "Windows" and self.link.is_dir():
+                for item in self.link.iterdir():
+                    if item.is_dir() and re.match(r"^\d+\.\d+$", item.name):
+                        if (item / "scripts").is_dir() or (item / "datafiles").is_dir():
+                            return item
```

Comment:
These other folders could also be used to check for valid places

```suggestion
            subpaths = ["config", "datafiles", "scripts", "extensions"]

            # Check for version folder (X.Y format) used by bforartists
            if get_platform() == "Windows" and self.link.is_dir():
                for item in self.link.iterdir():
                    if (
                        item.is_dir()
                        and re.match(r"^\d+\.\d+$", item.name)
                        and any((item / subpath).is_dir() for subpath in subpaths)
                    ):
                        return item
```